facade/disabledNodeFacade_test.go

```diff
@@ -0,0 +1,146 @@
+package facade
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDisabledNodeFacade_AllMethodes(t *testing.T) {
```

Comment:
add a `ShouldNotPanic` suffix to the test name?

Reply:
done